elastic_transport/_async_transport.py

```diff
@@ -208,8 +210,42 @@ async def perform_request(  # type: ignore[override,return]
         :arg retry_on_timeout: Set to true to retry after timeout errors.
         :arg request_timeout: Amount of time to wait for a response to fail with a timeout error.
         :arg client_meta: Extra client metadata key-value pairs to send in the client meta header.
+        :arg endpoint_id: The endpoint id of the request, such as `ml.close_job`.
+            Used for OpenTelemetry instrumentation.
+        :arg path_parths: Dictionary with all dynamic value in the url path.
```

Comment:
typo, s/parths/paths/

Reply:
Adressed in https://github.com/elastic/elastic-transport-python/pull/151/commits/bd6ae62550806e96691022def9a07aa929bf3386, thank you!